src/test/clojure/clara/test_rules.clj

```diff
@@ -3004,3 +3004,23 @@
 
       (is (= {:?w 10, :?t nil}
              (-> e (ex-data) :bindings))))))
```

Comment:
Please move this test to [test-accumulators](https://github.com/cerner/clara-rules/blob/master/src/test/common/clara/test_accumulators.cljc) and change it to use def-rules-test so that we'll run it in both Clojure and ClojureScript.  For reference, the test-accumulators namespace focuses on tests of individual accumulators, while test-accumulation focuses on the ability of Clara to run arbitrary accumulators.